### DISCUSSION TITLE
Menu too small - how to enlarge

### DISCUSSION THREAD

**Discussion by spinwitch** [freeplane/freeplane#1017] [Help] [answered]
Hi, I am very new to Linux (Mint Cinnammon 21.1 Vera) and have just installed freeplane. But it is impossible for me to do anything with it, because I simply cannot read the menu.
Here you can find a screenshot:  https://abload.de/img/bildschirmfotovom2023t2ios.png
(don't click to enlarge, that is NOT the way I see it)

I have a 17" monitor, my screen resolution in Linux is set to 125%, because I am slightly visually impaired.
Is there a way, to make the menu bar larger? I tried the preferences, but I can't read those either. Please help.

**Comment by macmarrum** [comment, marked as answer]
The setting you are looking for is `Preferences…->Appearance->Look and feel->Monitor size (inches)`.

But if you are unable to use Freeplane, you will need to change it directly in the configuration file, named `auto.properties`.
On Linux, the file is located in `$HOME/.config/freeplane/1.10.x/`. The part `1.10.x` corresponds to the version of Freeplane you are using, e.g. it could be `1.9.x` for the older series or `1.11.x` for the pre-release series.

Close your Freeplane. 
Go to `$HOME/.config/freeplane/1.10.x/` and open `auto.properties` in a text editor, like Gedit.
Search for a line starting with `monitor_size_inches=` and change the value to `17.0`.
Save and close the file.
Start your Freeplane.
If the menu is too large, change the value to something bigger, like `20.0`.
If it's too small, change it to a smaller value, like `15.0`.

**Reply by spinwitch** [reply]
Thank you very much. I used my magnifying glass, cause I don't trust myself with the console yet. I changed it to 16.0 and the size is fine now. And for a bonus - I made it dark, which makes me even more happy.
